spec/haskell/src/SEL4/Object/Reply.lhs

```diff
@@ -80,14 +80,13 @@ This module specifies the behavior of reply objects.
 >     when (nextReplyPtrOpt /= Nothing) $ do
 >         assert (isHead nextReplyPtrOpt) "the reply must be at the head"
 >         scPtr <- return $ theHeadScPtr nextReplyPtrOpt
->         tcbScPtrOpt <- threadGet tcbSchedContext tcbPtr
```

Comment:
spec update for replyPop, to be removed when @mktnk3's [PR](https://github.com/seL4/l4v/pull/185) is merged